app/views/posts/show.html.erb

```diff
@@ -1,2 +1,39 @@
-<h1> Single Posts </h1>
-<p>Content for single post here</p>
\ No newline at end of file
+<div class='show-post'>
+    <span class='title'>
+        Post by <%= @user.name %>
+    </span><br>
+        <p class="ici">
+            <span></span>
+            <span>comments : <span><% if (@post.comments_counter) %>
+            <%= @post.comments_counter %>
+            <% else %>
+            <%= 0 %>
+            <% end %>
+            , likes : <span><% if (@post.likes_counter) %>
+            <%= @post.likes_counter %>
+            <% else %>
+            <%= 0 %>
+            <% end %></span></span>
+            
+        </p>
+
+        <div>
+            <%= @post.text %>
+        </div>
+</div>
```

Comment:
- [ ] [OPTIONAL] I would recommend updating the comments counter according to the number of comments on the post.
- [ ] [OPTIONAL] According to the requirements the number of most recent comments under the post should be five but in the app I noticed only 3. I would suggest fixing the bug in the most_recent_comments methos. 
<img width="1070" alt="Screenshot 2023-10-07 at 8 06 34 PM" src="https://github.com/francksefu/WebApp/assets/44798044/f198f902-b867-429e-bceb-6d4ba9b68fa9">

- [ ] [OPTIONAL] I would also recommend updating the posts counter according to the number of posts of the user
- [ ] [OPTIONAL] In the wireframe the path for the page that shows all posts by the user is `user/:id/posts` however in the app the route remains `user/:id/` I would suggest fixing the bug in the routes.

<img width="1103" alt="Screenshot 2023-10-07 at 8 07 33 PM" src="https://github.com/francksefu/WebApp/assets/44798044/f08c316e-367b-491c-93cb-ae0def50ac21">